frontend/src/app/shared/components/input/input.component.sass

```diff
@@ -48,3 +48,9 @@ input:focus
     img
         margin-bottom: -4px
         width: 24px
+        height: 24px
+
+.hide img[src$='hide-1.svg']
```

Comment:
Don't use src here...
What it if will be changed
Create additional class if needed